api/v3/Payment.php

```diff
@@ -134,51 +134,72 @@ function civicrm_api3_payment_create(&$params) {
   }
   // Get contribution
   $contribution = civicrm_api3('Contribution', 'getsingle', array('id' => $params['contribution_id']));
-  if ($contribution['contribution_status'] != 'Partially paid') {
-    throw new API_Exception('Please select a contribution which has a partial payment');
+  $contributionStatus = CRM_Contribute_PseudoConstant::contributionStatus(NULL, 'name');
```

Comment:
This would be much clearer if you passed in the id - eg

```
$contributionStatus = CRM_Contribute_PseudoConstant::contributionStatus($contribution['contribution_status_id'], 'name');
```

@JoeMurray one for you - why check for pay later? If a payment is pending then it doesn't matter if they always meant to pay by check or whether they started to pay by credit card and then couldn't find one and then sent in a check.